model/symflower/symflower.go

```diff
@@ -35,9 +36,12 @@ var _ model.CapabilityWriteTests = (*Model)(nil)
 
 // generateTestsForFile generates test files for the given implementation file in a repository.
 func (m *Model) WriteTests(ctx model.Context) (assessment metrics.Assessments, err error) {
+	ctxWithTimeout, cancel := context.WithTimeout(context.Background(), language.DefaultExecutionTimeout)
```

Comment:
hmmmmmmmm not really sure about using the language timeout... cause symflower is a separate model. I.e. we might want to give more time to the SE than to the test execution.

How about adding a new value `SymbolicExecutionTimeout` to the `tools` package and a `Model.timeout` field to this model. That field can then be initialized with `tools.SymbolicExecutionTimeout` but overwritten for the tests.